ie11CustomProperties.js

```diff
@@ -514,6 +514,8 @@
 
 	var oldGetP = StyleProto.getPropertyValue;
 	StyleProto.getPropertyValue = function (property) {
+		if (arguments.length && !property) return '';
```

Comment:
I've had the same issue recently.
What is the reason for `arguments.length` check?
Also you could just call the original method: `return oldGetP.apply(this, arguments);`